Source/Scene/WebMapServiceImageryProvider.js

```diff
@@ -239,6 +239,37 @@ function WebMapServiceImageryProvider(options) {
         ? "EPSG:3857"
         : "CRS:84"
     );
+
+    // The axis order in previous versions of the WMS specifications was to always use easting (x or lon ) and northing (y or
+    // lat). WMS 1.3.0 specifies that, depending on the particular CRS, the x axis may or may not be oriented West-to-East,
+    // and the y axis may or may not be oriented South-to-North. The WMS portrayal operation shall account for axis order.
+    // This affects some of the EPSG codes that were commonly used such as ESPG:4326. The current implementation
+    // makes sure that coordinates passed to the server (as part of the GetMap BBOX parameter) as well as those advertised
+    // in the capabilities document reflect the inverse axe orders for EPSG codes between 4000 and 5000.
+    //  - Taken from Section 9.1.3 of https://download.osgeo.org/mapserver/docs/MapServer-56.pdf
+    var parts = parameters.crs.split(":");
+    if (parts[0] === "EPSG") {
+      var includesReverseAxis = [
```

Comment:
@tantricllama I guess these lookups could grow over time, it might be better housekeeping to keep them at the top of the file (rather than redefine them per invocation of `WebMapServiceImageryProvider`).